content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,361 @@
+
+   .. image:: lost_leads_email/example.png
+      :align: center
+      :alt: A lost leads reactivation filter list excluding lost reasons like Too Expensive.
+
+.. tip::
+   As filters are added and removed, pay attention to the :guilabel:`# record(s)` value below the
+   filtering section. This value indicates the total number of records that match the current
+   criteria.
+
+   To view a list of all matching records, click the :guilabel:`# record(s)` text.
+
+   .. image:: lost_leads_email/records.png
+      :align: center
+      :alt: The # record(s) text is located below the list of Recipient filters.
+
+.. _create_email:
+
+Minimum requirements
+====================
+
+In order to create and deliver a lost leads reactivation email campaign, the *CRM* and *Email
+Marketing* applications **must** be :doc:`installed <../../general/apps_modules>` and configured.
+
+Here are the minimum necessary filters that pertain to a lost leads reactivation mailing campaign:
+
+- The :ref:`Recipients <email_marketing/recipients_field>` field **must** be set to the
+  *Lead/Opportunity* model.
+- A :ref:`Blacklist <email_marketing/blacklist_filter>` filter to exclude unsubscribed recipients.
+- A :ref:`Created on <email_marketing/created_on_filter>` to target leads that were lost during a
+  specific period of time.
+- :ref:`Stage <email_marketing/stage_filter>` filter(s) to exclude leads that were already won, or
+  are still active in new stages of the sales pipeline (i.e. *New*, *Qualified*, etc.). These values
+  will be different per organization; however, it's minimally viable to exclude all the leads in the
+  *Won* stage.
+- One or more :ref:`Lost Reason <email_marketing/lost_reason_filter>` filters to exclude undesired
+  leads, such as duplicate, spam, or irrelevant records.
+- A pair of :ref:`Active <email_marketing/active_filter>` filters to target *both* active and
+  inactive leads.
+
+Add the necessary filters
+=========================
+
+First, navigate to the *Email Marketing* app, and on the :guilabel:`Mailings` page, click the
+:guilabel:`New` button in the top-left corner.
+
+.. _email_marketing/recipients_field:
+
+On the new :guilabel:`Mailings` form, enter an appropriate :guilabel:`Subject` line for the email in
+the corresponding field. Then, in the :guilabel:`Recipients` field, choose the
+:guilabel:`Lead/Opportunity` model from the drop-down menu.
+
+.. _email_marketing/blacklist_filter:
+
+In the rules section, located beneath the :guilabel:`Recipients` field, click the modify filter
+(:guilabel:`▶ (triangle pointing right)`) icon to expand the filter rules. Leave the default
+:guilabel:`Blacklist` rule in place.
+
+.. _email_marketing/created_on_filter:
+
+Created on
+----------
+
+Begin by clicking :guilabel:`New Rule` beneath the default :guilabel:`Blacklist` rule. Then, click
+the first field of the new rule that appears, and select the :guilabel:`Created on` parameter from
```

Comment:
Yea I see what you mean.
I think it's [arguably more "truthy" to say field](https://uxplanet.org/the-anatomy-of-input-field-c3ef863e01d7) since it's an *input field*, where *input* is the qualifying adjective to the noun *field*. Where it gets confusing, to me at least, is that the back-end terminology Odoo uses also matches the front-end in this case, since they're both "fields." 😅 
With that said, I'd like to keep it as is for now and see how it sits in the wild, with the hope that someone in the community would give us corrective feedback, if it's needed.